requirements_all.txt

```diff
@@ -190,6 +190,9 @@ hikvision==0.4
 # homeassistant.components.sensor.dht
 # http://github.com/adafruit/Adafruit_Python_DHT/archive/310c59b0293354d07d94375f1365f7b9b9110c7d.zip#Adafruit_DHT==1.3.0
 
+# homeassistant.components.light.mipow
+# https://github.com/mjg59/python-mipow/releases/download/0.3/mipow-0.3.tar.gz#mipow==0.3
```

Comment:
Do you have this dependency blocked from install by default because it won't build on certain platforms? Not necessarily an issue, just want to know + ensure this wasn't a mistake.